xpallets/gateway/records/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "xpallet-gateway-records"
-version = "1.0.0"
-authors = ["Chainpool <https://www.chainx.org>"]
+version = "2.0.0-alpha.0"
```

Comment:
```diff
diff --git a/Cargo.toml b/Cargo.toml
index 6cec2a8..b2bcc44 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,6 +33,8 @@ members = [
     "xpallets/assets/rpc",
     "xpallets/assets/rpc/runtime-api",
     "xpallets/gateway/bitcoin",
+    "xpallets/gateway/records",
     "xpallets/contracts",
     "xpallets/contracts/common",
     "xpallets/contracts/rpc",
```

The reason for package version is not bumped in #231 is it's not included in the workspace of `Cargo.toml`. We should add these missing packages into `workspace` section.

Reply:
Okay, I will add them into workspace.